optuna/storages/rdb/storage.py

```diff
@@ -1009,6 +1009,12 @@ def get_n_trials(self, study_id, state=None):
         self._commit(session)
         return n_trials
 
+    def read_from_remote_storage(self, study_id: int) -> None:
+        # Check that that study exists.
```

Comment:
How about changing this comment to the following?
```suggestion
        # Make sure that the given study exists.
```

Reply:
I adopted the change.